apps/passport-server/src/services/generic-issuance/pipelines/PretixPipeline.ts

```diff
@@ -1022,14 +1095,22 @@ export class PretixPipeline implements BasePipeline {
       );
 
       if (this.definition.options.enablePODTickets) {
-        const podTickets = await Promise.all(
-          tickets.map((ticket) => {
-            return this.getOrGenerateTicket<PODTicketPCD>(
-              ticket.claim.ticket,
-              PODTicketPCDTypeName
-            );
-          })
-        );
+        const podTickets = (
+          await Promise.all(
+            emails.map((e) =>
+              e.semaphoreV4Id
+                ? this.getPODTicketsForEmail(
+                    e.email,
+                    semaphoreId,
+                    e.semaphoreV4Id
+                  )
+                : undefined
+            )
+          )
+        )
+          .flat()
+          .filter((t) => t !== undefined) as PODTicketPCD[];
+
```

Comment:
this is where we use the v4 identity from the email pcd and stick it in the pod ticket